tests/scripts/nightly/run_wh_b0_only.sh

```diff
@@ -12,6 +12,26 @@ echo "Running nightly tests for WH B0 only"
 env pytest tests/ttnn/integration_tests/unet                # -> failing: issue #7556
 SLOW_MATMULS=1 WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml env pytest tests/ttnn/integration_tests/stable_diffusion
 
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_basic_transformer_block.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_cross_attention.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_cross_attn_up_block_2d.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_down_block_2d.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_downsample_2d.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_embedding.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_feedforward.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_geglu.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_perf_stable_diffusion.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_resnet_block_2d.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_sharded_attention.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_sharded_matmuls.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_transformer_2d_model.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_ttnn_cross_attention_down_block_2d.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_unet_2d_condition_model.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_unet_mid_block_2d_cross_attn.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_upblock_2d.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_upsample_2d.py
+env pytest tests/ttnn/integration_tests/stable_diffusion/test_upsample_nearest_2d.py
```

Comment:
These tests will need to be run with env vars:
`SLOW_MATMULS=1 WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml `

Reply:
@AleksKnezevic, will add these environment variables. Would it be enough to add the test in a single line? like
`SLOW_MATMULS=1 WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml env pytest tests/ttnn/integration_tests/stable_diffusion/tests/`

Reply:
yeah, that's fine. I believe there's already a `SLOW_MATMULS=1 WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml env pytest tests/ttnn/integration_tests/stable_diffusion` line just above these additions